tutorial/creating_packages/handle_sources/run_example.py

```diff
@@ -37,7 +37,7 @@
 output = run("conan list hello/1.0#:*")
 
 # assert that there are 3 binaries in the cache
-assert output.count("PID: ") == 3
```

Comment:
IMO, all these counts look a little bit fragile. What if we get the JSON file and read it? I think that could be better and more robust. WDYT?

Reply:
Yes I think that's much better, maybe we can merge this to have the ci in green and add some function to count packages in the near future?

Reply:
I added a TODO